src/routes/RemediationDetails.js

```diff
@@ -92,8 +113,43 @@ class RemediationDetails extends Component {
         return (status ? 'Enabled' : 'Disabled');
     }
 
+    renderLatestActivity = (playbookRuns) => {
+
+        if (playbookRuns.length) {
+
+            const mostRecent = playbookRuns[0];
+
+            return (
+                <FlexItem breakpointMods={ [{ modifier: FlexModifiers['spacer-xl'] }] }>
+                    <DescriptionList
+                        hasGutter
+                        title='Latest activity'>
+                        { renderStatusIcon(mostRecent.status) }
```

Comment:
This will receive one of these https://github.com/RedHatInsights/insights-remediations/blob/master/src/api/openapi.yaml#L623

Please update this for the icon to be returned based on a normalized status. Also, can we wrap the icon with a tooltip that shows non-normalized status text? E.g. if the playbook_run is in `pending` state you would see "running" icon with a tooltip text that clarifies is it currently pending?